go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/ONSdigital/dp-deployer
 go 1.13
 
 require (
+	github.com/ONSdigital/dp-api-clients-go v1.1.0
```

Comment:
Think we are on v1.9.0 now - worth updating to latest version:

`go get github.com/ONSdigital/dp-api-clients-go@latest` <- should be v1.9.0 at least
Then `go mod tidy` <- tidies up go.sum file